### PR TITLE
Order adapter domain 

### PR DESCRIPTION
**Pull request by Islom-35**
add order repo methods

### CODE REVIEW

File: internal/order/adapters/order_repo.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	return order, err
+}
+
+func (o *orderRepo) GetAllOrders() ([]domain.Order, error) {
+	rows, err := o.db.Query(`SELECT id, number, basket_id, total_price, status, created_at, updated_at FROM orders`)
+	if err != nil {
+		return nil, err
+	}
+
+	orders := make([]domain.Order, 0)
```
Comment (xbakhromjon) [inline internal/order/adapters/order_repo.go:42]: bu ishlmasligi mumkin. Tekshirib ko'ring. Initial capasity'ni 0 qilib append qilsa bo'ladimi yoki yo'qmi?

**Review by xbakhromjon — COMMENTED** [review]
Seems like norm

File: internal/order/domain/repository.go
Comment (asadbek21iman) [inline internal/order/domain/repository.go:1]: "defer rows.Close()" in GetAllOrders method

File: internal/order/adapters/order_repo.go (hunk 1 above)
Comment (Islom-35) [reply internal/order/adapters/order_repo.go:42]: bu aka bosh slice ochib beradi slice ga value qoshilganda ozi capasity ni kattalashtirib ketaverdi

File: internal/order/domain/repository.go
Comment (Islom-35) [reply internal/order/domain/repository.go:1]: ok

Comment (Islom-35) [reply internal/order/domain/repository.go:1]: I did it

**Review by khudayberdiyev25 — APPROVED** [review]

**Pull request merged**